src/Core/CMakeLists.txt

```diff
@@ -31,16 +31,21 @@ add_library(
     ${core_inlines}
     ${RA_VERSION_CPP}
 )
-addExternalFolder( Core ${CMAKE_CURRENT_SOURCE_DIR}/external )
+
+set(LocalDependencies)
+if (DEFINED Eigen3_DIR)
+    set(LocalDependencies "-DEigen3_DIR=${Eigen3_DIR}")
+endif ()
+addExternalFolder(Core ${CMAKE_CURRENT_SOURCE_DIR}/external ${LocalDependencies})
```

Comment:
don't get the point here, (but I do not try without) 
should Eigen3_DIR def be propagated to addExternalFolder automatically ?
Is it to give Eigen3_DIR to other dependencies ? if so how it is working currently without this def ?

Reply:
As `addExternalFolder` creates a cmake sub-process, the current cache is not propagated to the child process. So, we must pass explicitely somme additional args to this cmake child process. 
The macro `addExternalFolder` (i do not change the macro)  takes any number of parameters. The first two are used internally by the macro, the others are passed to the cmake child process. This is the only way I found to give the child cmake process (which configure and build the externals) informations about locally installed package that it might build or that it depends on.